markdown_lab/core/rust_backend.py

```
@@ -49,15 +49,15 @@ def convert_html_to_format(
     ) -> str:
         """
         Converts HTML content to the specified output format using the Rust backend.
-        
+
         Parameters:
             html (str): The HTML content to convert.
             base_url (str): The base URL used to resolve relative links in the HTML.
             output_format (str, optional): The desired output format ("markdown", "json", or "xml"). Defaults to "markdown".
```

Pylint (3.3.7)

[convention] Line too long (124/100)

(C0301)